lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -194,16 +194,27 @@ void StagesSubject::stageChanged(UsdNotice::ObjectsChanged const& notice, UsdSta
 		// don't care about it. In those cases, the changePath will contain something like:
 		//   "/<prim>.visibility"
 		//   "/<prim>.xformOp:translate"
-		if (!changedPath.IsPrimPath())
+		if (changedPath.IsPropertyPath())
 			continue;
 
-		const std::string& usdPrimPathStr = changedPath.GetPrimPath().GetString();
 		// Assume proxy shapes (and thus stages) cannot be instanced.  We can
 		// therefore map the stage to a single UFE path.  Lifting this
 		// restriction would mean sending one add or delete notification for
 		// each Maya Dag path instancing the proxy shape / stage.
-		Ufe::Path ufePath = stagePath(sender) + Ufe::PathSegment(usdPrimPathStr, g_USDRtid, '/');
-		auto prim = stage->GetPrimAtPath(changedPath);
+		Ufe::Path ufePath;
+		UsdPrim prim;
+		if (changedPath.GetString() == "/")
```

Comment:
How about we replace this with
```
changedPath == SdfPath::AbsoluteRootPath()
```

Reply:
There's also this:

```C++
if (changedPath.IsAbsoluteRootPath()) {
...
```

https://github.com/PixarAnimationStudios/USD/blob/d8a405a1344480f859f025c4f97085143efacb53/pxr/usd/sdf/path.h#L338

Reply:
Ahhh... that's what I was looking for. :)   Addressed in d109480e4277aac04786963e4e374cf3aa09a52c